symfit/core/fit.py

```diff
@@ -1243,7 +1235,8 @@ class Fit(HasCovarianceMatrix):
         fit_result = fit.execute(minimizer_kwargs=[dict(xatol=0.1), {}])
     """
 
-    @keywordonly(objective=None, minimizer=None, constraints=None)
+    @keywordonly(objective=None, minimizer=None, constraints=None,
+                 absolute_sigma=None)
```

Comment:
The default value for `absolut_sigma` should be True, rather than None

Reply:
Good suggestion as I first thought you were right, but it turns out that in `TakesData` it is actually also None, and then determined there. (Usually true by default indeed.)